index.html

```diff
@@ -21,174 +28,217 @@
-        </div>
-      </section>
-      <section class="project">
-        <div class="template">
-          <img src="icons/project3.svg" alt="Example image">
-        </div>
-        <div class="descr">
-          <h2>Project name goes here</h2>
-          <div class="lang">
-            <h3>HTML/CSS</h3>
-            <h3>Ruby on Rails</h3>
-            <h3>JavaScript</h3>
-          </div>
-          <button type="button">See This Project →</button>
-        </div>
-      </section>
-      <section class="project">
-        <div class="template">
-          <img src="icons/project4.svg" alt="Example image">
+      <div id ="main_project">
+        <div id="main_proj_left">
+          <h1 id="main_title">Projects</h1>
+          <img id="dots2" src="icons/dots_grid.svg" alt="dots">
         </div>
-        <div class="descr">
-          <h2>Project name goes here</h2>
-          <div class="lang">
-            <h3>HTML/CSS</h3>
-            <h3>Ruby on Rails</h3>
-            <h3>JavaScript</h3>
-          </div>
-          <button type="button">See This Project →</button>
-        </div>
-      </section>
-      <section class="project">
-        <div class="template">
-          <img src="icons/project5.svg" alt="Example image">
-        </div>
-        <div class="descr">
-          <h2>Project name goes here</h2>
-          <div class="lang">
-            <h3>HTML/CSS</h3>
-            <h3>Ruby on Rails</h3>
-            <h3>JavaScript</h3>
-          </div>
-          <button type="button">See This Project →</button>
-        </div>
-      </section>
-      <section class="project">
-        <div class="template">
-          <img src="icons/project6.svg" alt="Example image">
+        <section id="main_proj_right" class="project">
+          <div class="template">
+            <img class="img_proj1" src="icons/project1.svg" alt="Example image">
+          </div>
+          <div class="descr no_del">
+            <h2>Project name goes here</h2>
+            <div class="lang">
+              <h3>HTML/CSS</h3>
+              <h3>Ruby on Rails</h3>
+              <h3>JavaScript</h3>
+            </div>
+            <button class="btn2" type="button">See This Project →</button>
+          </div>
+        </section>
+      </div>
+      <div id="sec_proj">
+        <section id="grid1" class="project">
+          <div class="template">
+            <img class="img_left1" src="icons/project2.svg" alt="Example image">
+          </div>
+          <div class="descr del">
+            <h2>Project name goes here</h2>
+            <div class="lang">
+              <h3>HTML/CSS</h3>
+              <h3>Ruby on Rails</h3>
+              <h3>JavaScript</h3>
+            </div>
+            <button type="button">See This Project →</button>
+          </div>
+        </section>
+        <div id="grid2" class="project">
+          <div class="dt_template">
+            <img class="img_left2" src="icons/geometry.svg" alt="Example image">
+          </div>
```

Comment:
Refer to the [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9) template to fix this issue:

--------------------------------------------Yours----------------------------------------------

![port38](https://user-images.githubusercontent.com/55635977/130149826-50af8aa3-a932-4ef3-8320-911d3a388496.png)

--------------------------------------------Original----------------------------------------------

![port36](https://user-images.githubusercontent.com/55635977/130150177-bc439726-7799-4a18-bc55-1f0daedfc7d5.png)